### DISCUSSION TITLE
how to run the code of c programming i write in visual code?

### DISCUSSION THREAD

**Discussion by vegitoxxx** [microsoft/vscode-cpptools#10298] [General]
how to run the code of c programming i write in visual studio  code? 
I am new in this site/and in coding in general.
I am using visual studio 2022 enterpise
what is the similar compiler in visual studio code

**Comment by sean-mcmanus** [comment]
We have documentation on how to configure and build in VS Code using the Visual Studio 2022 C++ compiler at https://code.visualstudio.com/docs/cpp/config-msvc .
